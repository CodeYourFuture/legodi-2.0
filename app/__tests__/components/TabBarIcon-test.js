import React from 'react'
import renderer from 'react-test-renderer'
import TabBarIcon from '../../components/TabBarIcon'
import { Icon } from 'expo'

it('renders correctly', () => {
  const tree = renderer.create(<TabBarIcon><Icon.Ionicons /></TabBarIcon>).toJSON()

  expect(tree).toMatchSnapshot()
})
