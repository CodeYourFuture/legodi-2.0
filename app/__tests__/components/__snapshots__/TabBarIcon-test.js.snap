// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabBarIcon Component renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
/>
`;

exports[`renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
/>
`;
